### PR TITLE
Added common linters to camera_calibration_parsers

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: camera_calibration_parsers/src/camera_calibration_parsers/__init__.py [hunk 1]
```diff
@@ -1,9 +1,40 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+# All rights reserved.
+#
```
Comment (clalancette) [inline camera_calibration_parsers/src/camera_calibration_parsers/__init__.py]: This doesn't seem exactly right to me.  This file was originally contributed as part of commit 9829b02b48bae412a087e2183b3fd2c94fb16f26 in 2016.

Having no "Copyright" line at the beginning would be the most correct.  Do the linters succeed in that case?

(there are a few other instances of this pattern below)

Comment (ahcorde) [reply camera_calibration_parsers/src/camera_calibration_parsers/__init__.py]: maybe we can exclude these files from the copyright linter

Comment (ahcorde) [reply camera_calibration_parsers/src/camera_calibration_parsers/__init__.py]: friendly ping @clalancette

File: camera_calibration_parsers/src/camera_calibration_parsers/__init__.py [hunk 2]
```diff
@@ -1,9 +1,40 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
```
Comment (clalancette) [inline camera_calibration_parsers/src/camera_calibration_parsers/__init__.py]: In my testing, this works:

```suggestion
# Copyright 2016
```

File: camera_calibration_parsers/src/parse_wrapper.cpp [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
-*  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
-*  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
-*  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-*  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-*  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-*  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
-*  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-*  POSSIBILITY OF SUCH DAMAGE.
-*********************************************************************/
+// Copyright 2024 Open Source Robotics Foundation, Inc.
```
Comment (clalancette) [inline camera_calibration_parsers/src/parse_wrapper.cpp]:
```suggestion
// Copyright 2016
```

File: camera_calibration_parsers/test/parser.py [hunk 4]
```diff
@@ -1,48 +1,50 @@
 #!/usr/bin/env python
```
Comment (clalancette) [inline camera_calibration_parsers/test/parser.py]: While we are here, we can remove this line; it won't work on Linux anyway:

```suggestion
```

File: camera_calibration_parsers/src/parse_ini.cpp [hunk 5]
```diff
@@ -1,38 +1,33 @@
...
-*  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
-*  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-*  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-*  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-*  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
-*  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-*  POSSIBILITY OF SUCH DAMAGE.
-*********************************************************************/
+// Copyright (c) 2009, Willow Garage, Inc.
+// Copyright (c) 2018, Open Robotics
```
Comment (clalancette) [inline camera_calibration_parsers/src/parse_ini.cpp]: @mjcarroll You seemed to have added this line originally.  Any objections to us changing this to:

```suggestion
// Copyright (c) 2018, Open Source Robotics Foundation, Inc.
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: camera_calibration_parsers/src/parse_wrapper.cpp (hunk 3 above)
Comment (ahcorde) [reply camera_calibration_parsers/src/parse_wrapper.cpp]: https://github.com/ros-perception/image_common/pull/317/commits/a10661f39f06abb243803502b022ee8c368ce459

File: camera_calibration_parsers/src/camera_calibration_parsers/__init__.py (hunk 2 above)
Comment (ahcorde) [reply camera_calibration_parsers/src/camera_calibration_parsers/__init__.py]: https://github.com/ros-perception/image_common/pull/317/commits/a10661f39f06abb243803502b022ee8c368ce459

File: camera_calibration_parsers/src/parse_ini.cpp (hunk 5 above)
Comment (mjcarroll) [reply camera_calibration_parsers/src/parse_ini.cpp]: Nope, please feel free to update it.

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21533/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15817/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1028/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22250/)

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21540/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15824/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1035/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22257/)

**Pull request merged**